common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -97,7 +97,7 @@ div.problem {
     input[type="radio"],
     input[type="checkbox"] {
       @include float(left);
-      @include margin(4px, 8px, 0, 0);
+      // @include margin(4px, 8px, 0, 0);
```

Comment:
TODO: @frrrances, I was not able to figure out a way to have the Sass compile so that it was using our RTL ``margin()``mixin vs. Bourbon's. If you have ideas or want to try to pair on this for a few, let me know. I could use your help (**commenting this out is not good to merge**)